README.md

```diff
@@ -1 +1,42 @@
-# library
\ No newline at end of file
+# Library
+
+![image](./static/Screenshot.png)
+
+This is a Library Javascript project where user can dynamically add books and store in localstorage. User can toggle between read and not read button. user can delete and add a book.
+
```

Comment:
## Improve Readme

- [ ] Let's add a screenshot and instructions on how to well clone and run the project locally (setup). Let's keep it more descriptive enough ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).